COMPILING

```diff
@@ -36,7 +36,7 @@ We assume that you have a Debian/Ubuntu or Red Hat-like distribution.
 
    On Red Hat/Fedora or derivates, do
 
-   yum install gcc gcc-c++ flex bison perl-libwww-perl patch
+   yum install gcc gcc-c++ flex bison perl-libwww-perl patch devtoolset-6
```

Comment:
I think that when you use `devtoolset-6` you can remove some of dependencies. YUM should be fine with
```bash
yum install flex bison perl-libwww-perl patch devtoolset-6
```